src/containerapp/azext_containerapp/java_component_decorator.py

```diff
@@ -54,10 +54,20 @@ def get_argument_service_bindings(self):
     def get_argument_unbind_service_bindings(self):
         return self.get_param("unbind_service_bindings")
 
+    def get_argument_min_replicas(self):
+        return self.get_param("min_replicas")
+
+    def get_argument_max_replicas(self):
+        return self.get_param("max_replicas")
+
     def construct_payload(self):
         self.java_component_def["properties"]["componentType"] = self.get_argument_target_java_component_type()
         self.set_up_service_bindings()
         self.set_up_unbind_service_bindings()
+        self.java_component_def["properties"]["scale"] = {
+            "minReplicas": self.get_argument_min_replicas(),
+            "maxReplicas": self.get_argument_max_replicas()
+        }
```

Comment:
I notice this `construct_payload` used by `create` and `update` both. But for `update` command, we should not set `None` to it if customer didn't input argument `--min-replicas` or `--max-replicas`, right?

Reply:
Good catch, thx!